stylus/include/AboutUs.styl

```diff
@@ -0,0 +1,44 @@
+.AboutUs
+	>div:nth-child(1)
+		display: flex
+		padding-bottom: 6rem
+		padding-top: 4rem
+		>div
+			width: 50%
+			>img
+				padding-left: 3rem
+				padding-right: 3rem
+			>div
+				margin-right: 5rem
+			>div:nth-child(1)
+				font-family: Arvo
+				font-size: 2rem
+			>div:nth-child(8)
+				font-weight: bold
+				text-transform: uppercase
+		>div:nth-child(2)
+			padding-top: 3rem
+	>div:nth-child(2)
+		display: flex
+		flex-wrap: wrap
+		justify-content: center
+		.calendar-image
+			background: $Mustard
+			flex: 1
+			margin: 1rem
+			position: relative
+			>div
+				background: $Charcoal
+				color: white
+				font-family: Arvo
+				left: 50%
+				padding: 1rem
+				position: absolute
+				text-align: center
+				top: 50%
+				transform: translate(-50%, -50%)
+		.calendar-image:hover
```

Comment:
Instead of repeating the class name, you can indent and use the parent selector, `&:hover`. Note that this is not valid css but a convenience offered by the preprocessor (stylus). I use it all the time.